deployments/rubintv/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: events
+
+resources:
+  - github.com/lsst-sqre/rubintv.git//manifests/base?ref=0.0.1
```

Comment:
Just make sure to create this `0.0.1` tag before deploying.

Reply:
Also, is the repo name `rubin-tv`?

Reply:
Oh, you're right.  The repo name has the hyphen.